src/event.ts

```diff
@@ -90,11 +90,11 @@ interface EventStatusRemove extends FieldsTargeted {
 	status: number
 }
 
-/** The server has snapshot the effects of an action on its target. */
-interface EventSnapshot extends FieldsTargeted {
+/** The server has confirmed the execution of an action on its target. */
+interface EventExecute extends FieldsTargeted {
 	/** XIV Action ID */
 	action: number
```

Comment:
hm. wonder if we need the action id here any more - it would need to be fabricated on any event source that isn't matching them up for us.

Reply:
My thinking is to keep it for now, and if we run into it being undesirable with a future event source we can nuke it then.